### PR TITLE
don't let oTracking ruin it for everyone

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: src/o-tracking-wrapper.js [hunk 1]
```diff
@@ -9,19 +11,27 @@ const oTrackingWrapper = {
...
+					layout: oGrid.getCurrentLayout(),
+					orientation: oViewport.getOrientation()
+				}
+			});
+
+			// The page url and referrer automatically populated by o-tracking
+			oTracking.page({});
+		}catch(err){
+			if(typeof console === 'object' && typeof console.error === 'function'){
+				console.error(err);
```
Comment (matthew-andrews) [inline src/o-tracking-wrapper.js]: possible to use o-errors in this case?

Comment (matthew-andrews) [inline src/o-tracking-wrapper.js]: or does that use o-tracking undr the hood?

Comment (aintgoin2goa) [inline src/o-tracking-wrapper.js]: No it doesn't, but is complicated as oErrors is busy getting initialized elsewhere in the code - but I could pass the instance in and use that

**Comment by adambraimbridge** [comment]
:+1:

**Pull request merged**